### PR TITLE
feat(client): remove extra if statement in the navigation header

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

in line 73 of universal-nav, there is an if statement that stop `NavLinks` from rendering while the using it pending, so asking the `NavLinks` to wait is redundant.

https://github.com/Sboonny/freeCodeCamp/blob/4f522f4d7541f2adc608f508c3163fecbb93e231/client/src/components/Header/components/universal-nav.tsx#L73

### CODE REVIEW

File: client/src/components/Header/components/nav-links.tsx [hunk 1]
```diff
@@ -33,7 +33,6 @@ const locales = availableLangs.client.filter(
 export interface NavLinksProps extends Pick<ThemeProps, 'toggleNightMode'> {
   displayMenu: boolean;
```
Comment (smithbinky) [inline client/src/components/Header/components/nav-links.tsx:34]: salvation army in Denver Colorado.

Comment (smithbinky) [reply client/src/components/Header/components/nav-links.tsx:34]: eng_usa.

Comment (smithbinky) [reply client/src/components/Header/components/nav-links.tsx:34]: Ca

Comment (smithbinky) [reply client/src/components/Header/components/nav-links.tsx:34]: toomph from_notical_star.

Comment (smithbinky) [reply client/src/components/Header/components/nav-links.tsx:34]: CA, DTLA, Bonaventure.

Comment (smithbinky) [reply client/src/components/Header/components/nav-links.tsx:34]: Marina del Rey Ritz Carlton.

Comment (smithbinky) [reply client/src/components/Header/components/nav-links.tsx:34]: UDR leasing 717 Olympic.

Comment (smithbinky) [reply client/src/components/Header/components/nav-links.tsx:34]: ape

**Review by smithbinky — COMMENTED** [review]
sje, smith James e.

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**